Library/Homebrew/dev-cmd/sh.rb

```diff
@@ -39,32 +39,25 @@ def run
 
         ENV["VERBOSE"] = "1" if args.verbose?
 
-        preferred_shell = Utils::Shell.preferred_path(default: "/bin/bash")
+        preferred_path = Utils::Shell.preferred_path(default: "/bin/bash")
 
         if args.cmd.present?
-          safe_system(preferred_shell, "-c", args.cmd)
+          safe_system(preferred_path, "-c", args.cmd)
         elsif args.named.present?
-          safe_system(preferred_shell, args.named.first)
+          safe_system(preferred_path, args.named.first)
         else
-          shell_type = Utils::Shell.preferred
-          subshell = case shell_type
-          when :zsh
-            "PS1='brew %B%F{green}%~%f%b$ ' #{preferred_shell} -d -f"
-          when :bash
-            "PS1=\"brew \\[\\033[1;32m\\]\\w\\[\\033[0m\\]$ \" #{preferred_shell} --noprofile --norc"
```

Comment:
Any reason we're dropping `--noprofile --norc` (from https://github.com/Homebrew/brew/pull/9260)?

Reply:
@Bo98 Yes:
- with `brew sh` these seem to predate environment filtering so aren't super necessary
- with `brew bundle sh` it's nicer to include these

I did a bunch of local testing and these were the options that seemed to make the most sense. If they break workflows: we can change back.